apphosting.mainnet.yaml

```
@@ -39,6 +39,8 @@ env:
     value: southeastasia
   - variable: AZURE_SUBSCRIPTION_KEY
     secret: AZURE_SUBSCRIPTION_KEY
+  - variable: IS_MAINTENANCE_MODE
+    value: FALSE
```

```suggestion
  - variable: IS_MAINTENANCE_MODE
    value:
```
Or just remove the key if yaml complains